<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneDimensionalWindow</class>
 <widget class="QMainWindow" name="OneDimensionalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneDimensionalWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,5">
    <item row="0" column="1">
     <widget class="QTableView" name="celluralTable"/>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Liczba komórek</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="cellCountInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="drawButton">
        <property name="text">
         <string>Rysuj</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Liczba kroków symulacji</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="simulationStepsInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Reguła</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="ruleInput">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Rozpocznij symulację</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Liczba komórek ze stanem wysokim</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="highStateCountInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cellCountInput</tabstop>
  <tabstop>highStateCountInput</tabstop>
  <tabstop>drawButton</tabstop>
  <tabstop>simulationStepsInput</tabstop>
  <tabstop>ruleInput</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>celluralTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
